src/components/shooting/ShootingView.tsx

```diff
@@ -0,0 +1,273 @@
+'use client';
+
+import { useEffect, useRef, useState } from 'react';
+
+import {
+  Dialog,
+  DialogClose,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+} from '@/components/ui/dialog';
+import type { Avatar } from '@/lib/shooting/avatar';
+import { createTextTexture } from '@/lib/shooting/createTextTexture';
+import { DeviceOrientationControls } from '@/lib/shooting/DeviceOrientationControls';
+import * as TWEEN from '@tweenjs/tween.js';
+import * as Three from 'three';
+import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls.js';
+import { Button } from '../ui/button';
```

Comment:
`'../ui/button'` -> `'@/components/ui/button';`

```suggestion
import { Button } from '@/components/ui/button';
import {
  Dialog,
  DialogClose,
  DialogContent,
  DialogDescription,
  DialogFooter,
  DialogHeader,
  DialogTitle,
} from '@/components/ui/dialog';
import type { Avatar } from '@/lib/shooting/avatar';
import { createTextTexture } from '@/lib/shooting/createTextTexture';
import { DeviceOrientationControls } from '@/lib/shooting/DeviceOrientationControls';
import * as TWEEN from '@tweenjs/tween.js';
import * as Three from 'three';
import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls.js';
```